enrolment-ui/src/modules/Applications/ApplicationRequests.js

```diff
@@ -48,8 +48,8 @@ const ApplicationRequests = props => {
         <tbody>
           {!projects.loading &&
             !applications.loading &&
-            projects.data.results.map(project => {
-              const application = _.find(applications.data.results, { project: project.id });
```

Comment:
Dependency changes in the API make the response be directly in data, rather than in a nested returns attr.